historical_surveillance/forms.py

```diff
@@ -0,0 +1,27 @@
+from django import forms
+
+from .models import School, District, AggregateEnrollment, Enrollment
+
+
+class AggregateEnrollmentForms(forms.ModelForm):
+    class Meta:
+        model = AggregateEnrollment
+        fields = "__all__"
```

Comment:
See some comments below - but we can add required fields and additional validation here in the form instead of the view.